chart/values.yaml

```
@@ -38,7 +38,13 @@ remoteControlPlane:
     protocol: "http"                         # Used Protocol (http, https)
     apiValidateTls: true                     # Defines if the control plane certificate should be validated
     hostname: "api-gateway-nginx.keptn"      # Hostname of the control plane cluster, Port
-    token: ""                                # Keptn API Token
+    authMode: "token"                        # Flag to choose the defined authentication method, valid values are token and oauth
+    token: ""                                # Keptn API Token (leave empty if OAuth should be used)
```

I think that the comment is no longer relevant...
```suggestion
    token: ""                                # Keptn API Token
```